components/motor/gpio/motor_encoder.go

```diff
@@ -354,21 +354,28 @@ func (m *EncodedMotor) rpmMonitorPass(pos, lastPos, now, lastTime int64, rpmDebu
 	var ticksLeft int64
 
 	currentRPM := m.computeRPM(pos, lastPos, now, lastTime)
+	m.state.currentRPM = currentRPM
 
 	if !m.state.regulated && math.Abs(m.state.desiredRPM) > 0.001 {
 		m.rpmMonitorPassSetRpmInLock(currentRPM, m.state.desiredRPM, -1, rpmDebug)
 		return
 	}
 
 	if m.state.regulated {
-		ticksLeft = (m.state.setPoint - pos) * sign(m.state.lastPowerPct)
+		ticksLeft = (m.state.setPoint - pos) * sign(m.state.lastPowerPct) * m.flip // should always be positive
```

Comment:
It's not obvious to me that the comment reflects what the code is doing. If this should always be positive, should we put it in a `math.abs()` call?

Reply:
Test is failing, I have another commit with more fake motor logic that I'm cleaning up. The test file will look like this however.